### PR TITLE
REDDEV-749 Upgrade to Bootstrap 5

### PR DESCRIPTION
**Pull request by heathharrelson**
Make changes needed to upgrade to Bootstrap 5, introduced in REDCap 13.4.0:

* Use `data-bs-` attributes for JS integration
* Fix styling of form `<select>` inputs
* Fix styling of calendar input addons
* Bump minimum REDCap version

localhost_redcap_redcap_v13 9 2_ExternalModules__prefix=vizr page=index pid=17 (1)

### CODE REVIEW

File: lib/metadata.php [hunk 1]
```diff
@@ -42,11 +42,6 @@
 $metadata_object->recordIdField = $record_id_field;
 $metadata_object->dataDictionary = $data_dictionary;
 $metadata_object->events = $event_map;
-if (REDCap::versionCompare ( REDCAP_VERSION , '8.7.1', '<')) {
-	$metadata_object->bootstrapVersion = 3;
-} else {
-	$metadata_object->bootstrapVersion = 4;
-}
```
Comment (heathharrelson) [inline lib/metadata.php:49]: This is dead code. The only references to `bootstrapVersion` are in this file.

File: config.json [hunk 2]
```diff
@@ -35,7 +35,7 @@
     }
   ],
   "compatibility": {
-      "redcap-version-min": "8.5.4",
+      "redcap-version-min": "13.4.0",
```
Comment (heathharrelson) [inline config.json:38]: This is the version where Bootstrap 5 was introduced into REDCap.

**Review by aeyates — APPROVED** [review]
I'm glad the changes were pretty minimal. Hopefully it didn't take long once you identified the issue.

**Comment by heathharrelson** [comment]
> I'm glad the changes were pretty minimal. Hopefully it didn't take long once you identified the issue.

Identifying the necessary changes was very easy. Most of the time was actually spent setting up a local instance of REDCap and reviewing the code, so I would be able to speak intelligently when I worked through the changes with @saligiad.

**Pull request merged**